core/templates/domain/improvements/high-bounce-rate-task.model.spec.ts

```diff
@@ -254,27 +265,27 @@ describe('High bounce rate task', function() {
           },
         }));
       expect(() => task.refreshStatus(
-        statsWithWrongVersion, 1, this.config)).toThrowError(
+        statsWithWrongVersion, 1, config)).toThrowError(
```

Comment:
This should be formatted similarly to lines 243 to 247

Reply:
done... for some reason, I keep getting the formatting wrong. Is there a specific formatter that needs to be used?

Reply:
We don't really use any formatter right now, and the formatting is a bit subjective. I think here the main problem are the lies below, where you had
```
'aaaa' +
  'bbbb')
```
but the problem is that these need to have equal indentation since the are on the same level.